enterprise/cmd/frontend/internal/authz/resolvers/permissions_sync_jobs.go

```diff
@@ -0,0 +1,67 @@
+package resolvers
+
+import (
+	"github.com/graph-gophers/graphql-go"
+	"github.com/graph-gophers/graphql-go/relay"
+
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/graphqlbackend"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/graphqlbackend/graphqlutil"
+	"github.com/sourcegraph/sourcegraph/enterprise/internal/authz/syncjobs"
+	"github.com/sourcegraph/sourcegraph/internal/gqlutil"
+)
+
+type permissionsSyncJobsConnection struct {
+	jobs []graphqlbackend.PermissionsSyncJobResolver
+}
+
+var _ graphqlbackend.PermissionsSyncJobsConnection = &permissionsSyncJobsConnection{}
+
+func (r *permissionsSyncJobsConnection) Nodes() []graphqlbackend.PermissionsSyncJobResolver {
+	return r.jobs
+}
+
+// We don't yet support pagination, but we have the fields for future-compat
+func (r *permissionsSyncJobsConnection) TotalCount() int32 { return int32(len(r.jobs)) }
+func (r *permissionsSyncJobsConnection) PageInfo() *graphqlutil.PageInfo {
+	return graphqlutil.HasNextPage(false)
+}
+
+type permissionsSyncJobResolver struct{ s syncjobs.Status }
+
+var _ graphqlbackend.PermissionsSyncJobResolver = permissionsSyncJobResolver{}
+
+func (j permissionsSyncJobResolver) ID() graphql.ID {
+	// Use eventTime because RequestID can repeat
+	return relay.MarshalID("EventTime", j.s.Completed)
```

Comment:
The Kind is usually expected to be the name of the entity, otherwise this might still not be globally unique (what if someone adds the EventTime node type)

```suggestion
	// Use eventTime because RequestID can repeat
	return relay.MarshalID("PermissionsSyncJob", j.s.Completed)
```